### PR TITLE
Feature/archive

### PR DESCRIPTION
**Pull request by danalexilewis**
add in our old financial model with some links to it.

### CODE REVIEW

File: archive/readme.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Archive
+
+Here's anything we've done before and interated on:
```
Comment (ahdinosaur) [inline archive/readme.md]: s/interated/iterated

File: archive/readme.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+
+## Agreements
+ - [Financial V01](archive/financial-agreement-v01.md)
```
Comment (ahdinosaur) [inline archive/readme.md]: why `01` and not `1` or `1.0` or `1.0.0`?

File: archive/financial-agreement-v01.md [hunk 3]
```diff
@@ -0,0 +1 @@
+{% include "git+https://github.com/enspiral-root-systems/agreement-financial.git/README.md#4fe123a6fd67e3553f9934635677b2180af04baa" %}
```
Comment (ahdinosaur) [inline archive/financial-agreement-v01.md]: nice! :sparkles:

**Review by ahdinosaur — APPROVED** [review]
this is sweet. gave some minor comments.

File: archive/readme.md (hunk 2 above)
Comment (danalexilewis) [reply archive/readme.md]: i didn't give it that much thought - should probably be  1.0.1 as thats what it is tagged as

**Comment by danalexilewis** [comment]
@ahdinosaur done

**Comment by ahdinosaur** [comment]
oops, we forgot to add "Archives" to the main home page.

**Pull request merged**